<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="characterImagePlace">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>471</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/backgrounds/resources/images/backgrounds/filler.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>475</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="firststageLayout">
    <item>
     <layout class="QHBoxLayout" name="nameLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="firstchooseLayout">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="raceLayout">
        <item>
         <widget class="QLabel" name="raseLabel">
          <property name="text">
           <string>Раса:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="raceChoose">
          <item>
           <property name="text">
            <string>Человек</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Эльф</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Дворф</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="classLayout">
        <item>
         <widget class="QLabel" name="classLabel">
          <property name="text">
           <string>Класс:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="classChoose">
          <item>
           <property name="text">
            <string>Воин</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Разбойник</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Маг</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Жрец</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="storyLayout">
        <item>
         <widget class="QLabel" name="storyLabel">
          <property name="text">
           <string>Предистория:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="storyChoose">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <item>
           <property name="text">
            <string>Солдат</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ремесленник</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Артист</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Отшельник</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="constitutionLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>150</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Телосложение:  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constitutionCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>149</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="specsBackground">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>140</y>
     <width>371</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/backgrounds/resources/images/backgrounds/specsBackground.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="streightCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>189</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="strengthLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>189</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Сила:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dexterityLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>250</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ловкость:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dexterityCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>250</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="wisdomLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>379</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Мудрость:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wisdomCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>379</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intelligenceCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>510</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="charismaLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>670</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Харизма:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="charismaCount">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>670</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="intelligenceLabel">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>510</y>
     <width>59</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Интелект:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>189</y>
     <width>181</width>
     <height>555</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="skillsLayout">
    <item>
     <layout class="QVBoxLayout" name="skillsNamesLayout">
      <item>
       <widget class="QLabel" name="athleticsLabel">
        <property name="text">
         <string>Атлетика:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="acrobaticsLabel">
        <property name="text">
         <string>Акробатика:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sleightofhandLabel">
        <property name="text">
         <string>Ловкость рук:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stealthLabel">
        <property name="text">
         <string>Скрытность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="perceptionLabel">
        <property name="text">
         <string>Внимательность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="survivalLabel">
        <property name="text">
         <string>Выживание:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="medicineLabel">
        <property name="text">
         <string>Медицина:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="insightLabel">
        <property name="text">
         <string>Проницательность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="animalhandlingLabel">
        <property name="text">
         <string>Уход за животными:  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="investigationLabel">
        <property name="text">
         <string>Анализ:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="historyLabel">
        <property name="text">
         <string>История:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="arcanaLabel">
        <property name="text">
         <string>Магия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="natureLabel">
        <property name="text">
         <string>Природа:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="religionLabel">
        <property name="text">
         <string>Религия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="performanceLabel">
        <property name="text">
         <string>Выступление:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="intimitationLabel">
        <property name="text">
         <string>Запугивание:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deceptionLabel">
        <property name="text">
         <string>Обман:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="persuationLabel">
        <property name="text">
         <string>Убеждение:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="skillsCountsLayuot">
      <item>
       <widget class="QLineEdit" name="atlethicsCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="acrobaticsCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sleightofhandCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stealthCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="perceptionCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="survivalCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="medicineCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="insightCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="animalhandlingCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="investigationCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="historyCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="arcaneCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="natureCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="religionCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="performanceCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intimitationCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deceptionCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="persuationCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>150</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="skillpointLayout">
    <item>
     <widget class="QLabel" name="skillpointLabel">
      <property name="text">
       <string>Доступно очков:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="skillpointCount">
      <property name="text">
       <string>25</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="debuggingLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>debug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skillButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Подсчитать
 характеристики</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>specsBackground</zorder>
  <zorder>characterImagePlace</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>constitutionLabel</zorder>
  <zorder>constitutionCount</zorder>
  <zorder>streightCount</zorder>
  <zorder>strengthLabel</zorder>
  <zorder>dexterityLabel</zorder>
  <zorder>dexterityCount</zorder>
  <zorder>wisdomLabel</zorder>
  <zorder>wisdomCount</zorder>
  <zorder>intelligenceCount</zorder>
  <zorder>charismaLabel</zorder>
  <zorder>charismaCount</zorder>
  <zorder>intelligenceLabel</zorder>
  <zorder>debuggingLabel</zorder>
  <zorder>skillButton</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
